src/Features/Core/Portable/GenerateFromMembers/AbstractGenerateFromMembersService.cs

```diff
@@ -132,19 +116,11 @@ where constructorTypes.SequenceEqual(symbolTypes)
             return q.FirstOrDefault();
         }
 
-        protected bool HasMatchingConstructor(
-            INamedTypeSymbol containingType,
-            List<IParameterSymbol> parameters)
-        {
-            return containingType.InstanceConstructors.Any(c => MatchesConstructor(c, parameters));
-        }
+        protected IMethodSymbol GetMatchingConstructor(INamedTypeSymbol containingType, ImmutableArray<IParameterSymbol> parameters)
+            => containingType.InstanceConstructors.FirstOrDefault(c => MatchesConstructor(c, parameters));
```

Comment:
Maybe explain that the FirstOrDefault is to pick between identical constructors in error code.